.github/workflows/interface-unit-tests.yml

```diff
@@ -465,18 +465,12 @@ jobs:
       pytest_markers: external
       pytest_additional_args: -W ${{ inputs.python_warning_level }} ${{ inputs.python_warning_level != 'default' && '--continue-on-collection-errors' || '' }}
       additional_pip_packages: |
-        pyzx matplotlib stim quimb==1.8.4 mitiq ply optax
+        pyzx matplotlib stim quimb==1.8.4 mitiq ply optax scipy-openblas32>=0.3.26
```

Comment:
Does lightning/catalyst not define this dep and version?

Reply:
Due to how uv's dependency resolution works, it will only search across multiple indexes if you set the unsafe-best-match resolution strategy, which I wanted to avoid.

I added this here mainly so we are saying that this package has to be installed from pypi without going the unsafe-match route.

Now, there is likely-hood we can fix this if we on-board to pyproject.toml for all of pennylane deps as it is a lot more flexible on specifying sources of different deps, but I did not want to bring that overhaul into scope of this PR.

Reply:
Sounds good thanks. Yea, PEP517++ compliance here sounds like the better long term plan. Fine for now :+1: